### ISSUE TITLE
google_compute_instance pricing not working

### ISSUE THREAD

**Issue by amontalban** [infracost/infracost#410]
Hi team,

I'm trying to use this tool at my workplace (We are a GCP shop), but for some reason, I can't get the `google_compute_instance` pricing to work. Below is a simple test I used:

```tf
resource "google_compute_instance" "test" {
  name = "test"
  machine_type = "f1-micro"

  boot_disk {
    auto_delete = true
    initialize_params {
      image  = "https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-2004-focal-v20200729"
    }
  }

  network_interface {
    network = "default"
    access_config {
    }
  }
  scheduling {
    on_host_maintenance = "MIGRATE"
  }
  metadata = {
    ssh-keys = "ubuntu:${file("~/.ssh/id_rsa.pub")}"
  }
}
```

The output is:

```bash
(deployments) amontalban@Andress-MBP test % infracost --no-color
Loading resources from current directory
  ✔ Running terraform plan
  ✔ Running terraform show
✔ Calculating cost estimate

  NAME                                           MONTHLY QTY  UNIT   PRICE   HOURLY COST  MONTHLY COST  

  google_compute_instance.test                                                                          
  ├─ Linux/UNIX usage (on-demand, f1-micro)              730  hours  0.0000       0.0000        0.0000  
  └─ Standard provisioned storage (pd-standard)           10  GiB    0.0000       0.0000        0.0000  
  Total                                                                           0.0000        0.0000  
                                                                                                        
  OVERALL TOTAL (USD)                                                             0.0000        0.0000  
```

This is happening with multiple machine types, for example, I have tested with `n2-standard-4` and `n2-standard-16` with the same result.

Thank you for the help!

**Comment by aliscott** [comment]
Thanks @amontalban. This looks like an issue with it working out the correct region to use. I've fixed it in https://github.com/infracost/infracost/pull/411.

We'll push this in the next patch release shortly, but to unblock yourself just now if you set something like `zone = us-central1-a` explicitly in the resource then it should work.

**Comment by amontalban** [comment]
w00t w00t!! That fixed the issue.

Thanks for the quick reply and fix @aliscott!

**Comment by aliscott** [comment]
@amontalban the fix for this has now been released in v0.7.18.
